SUTrivBot/Repo/triviaQAData.json

```diff
@@ -62,9 +62,9 @@
     {
       "questionText": "What is Steven's canonical birth month? Bonus: name the day too",
       "answerText": "2017-08-15T04:00:00.000Z",
-      "answers": ["August"],
+      "answers": ["August", "2017-08-15T04:00:00.000Z"],
       "answersRequired": 1,
-      "bonusAnswers": ["August", "15"],
+      "bonusAnswers": ["August", "15", "2017-08-15T04:00:00.000Z"],
       "bonusAll": true,
```

Comment:
This is stating that to get the bonus answer requires that time stamp written exactly as is. 
However, the bonus text just says name the day too so the timestamp isn't needed at all. Just the month and date.